bokeh/models/graphs.py

```diff
@@ -54,8 +54,8 @@ def from_networkx(graph, layout_function, **kwargs):
         from ..models.renderers import GraphRenderer
         from ..models.graphs import StaticLayoutProvider
 
-        nodes = graph.nodes()
-        edges = graph.edges()
+        nodes = dict(graph.nodes())
```

Comment:
In order to support both <2 and >=2 versions, it's likely necessary to do a networkx version check then handle each major version appropriately.

Reply:
I agree.  Just wanted to get things going

Reply:
If this can be done with type checking e.g. `isinstance(graph.nodes(), dict)` I think that's preferable than an explicit version check.

Reply:
@bryevdv Yeah.  That's better.
BTW: Did you see my stackoverflow [question](https://stackoverflow.com/questions/46397671/using-bokeh-how-does-one-plot-variable-size-nodes-and-node-colors)?

Reply:
Something like this?

```
# Handle nx 1.x vs 2.x data structure change
if isinstance(graph.nodes(), dict):
    nodes = graph.nodes()
    edges = graph.edges()
else: 
    nodes = dict(graph.nodes())
    edges = dict(graph.edges())
```